packages/eslint-plugin-gestalt/src/helpers/styleValidators.js

```diff
@@ -40,6 +40,17 @@ const borderMap = {
   '2px lightgray solid': `borderStyle="lg"`,
 };
 
+const flexMap = {
+  '1 1 auto': 'grow',
+  '0 1 auto': 'shrink',
+  '0 0 auto': 'none',
+};
+
+type ToCamelCaseType = ({| attribute: string |}) => string;
+
+export const toCamelCase: ToCamelCaseType = ({ attribute }) =>
```

Comment:
nit: this is more specifically "kebab-case to camelCase", right? The name should be more specific so it doesn't appear to be agnostic about the initial case